### PR TITLE
Implementation of Cookie persistance + Setting in UI

### PR DESCRIPTION
**Pull request by PoisonousJohn**
Hey, you can find "Save cookies" under the "Privacy" section.

### CODE REVIEW

File: src/ui/Main.qml [hunk 1]
```diff
@@ -23,13 +23,18 @@
 
 import QtQuick 2.0
 import QtWebEngine 1.1
+import Qt.labs.platform 1.0
```
Comment (timsueberkrueb) [inline src/ui/Main.qml]: What's this import for?

File: src/ui/tabview/content/SettingsContent.qml [hunk 2]
```diff
@@ -329,6 +329,26 @@ TabContent {
...
+                    text: qsTr("Privacy")
+                }
+
+                Column {
+                    CheckBox {
+                        text: qsTr("Save cookies")
+                        checked: Settings.startConfig.persistentCookies
+                        onCheckedChanged: {
+                            if (checked === Settings.startConfig.persistentCookies)
+                            {
```
Comment (timsueberkrueb) [inline src/ui/tabview/content/SettingsContent.qml]: Curly brace style ;)

**Review by timsueberkrueb — CHANGES_REQUESTED** [review]
Thanks! Yet have to try this locally.

File: src/ui/Main.qml (hunk 1 above)
Comment (PoisonousJohn) [reply src/ui/Main.qml]: thx, not needed

**Comment by PoisonousJohn** [comment]
@tim-sueberkrueb  done

**Review by timsueberkrueb — APPROVED** [review]

**Comment by timsueberkrueb** [comment]
@PoisonousJohn can you add the cookies option to `defaultJSON` and make it `true` by default?

**Comment by timsueberkrueb** [comment]
Nevermind, went ahead and fixed that.

**Comment by timsueberkrueb** [comment]
Thank you!

**Pull request merged**